pyproject.toml

```diff
@@ -25,15 +25,15 @@ classifiers = [
     "Programming Language :: Python :: 3",
 ]
 dependencies = [
-    "Sphinx>=4.2.0,<6",
+    "Sphinx>=4.2.0",
```

Comment:
Did you check this is not causing any issue in the top navigation bar, @Revathyvenugopal162? I remember we imposed this condition due to this issue.

Reply:
Yes, it is fixed, i checked it. https://github.com/pydata/pydata-sphinx-theme/issues/1130 issue in pydata related to that.